keyboards/darkproject/kd87b_ltd/info.json

```
@@ -0,0 +1,279 @@
+                { "label": ".", "matrix": [7, 4], "x": 10.25, "y": 4.5 },
+                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 4.5 },
+                { "label": "Shift", "matrix": [9, 1], "x": 12.25, "y": 4.5, "w": 1.75},
+                { "label": "Up", "matrix": [3, 5], "x": 16.25, "y": 4.5 },
+
+                { "label": "Ctrl", "matrix": [0, 6], "x": 0, "y": 5.5, "w": 1.25},
+                { "label": "Win", "matrix": [9, 0], "x": 1.25, "y": 5.5, "w": 1.25},
+                { "label": "Alt", "matrix": [9, 3], "x": 2.5, "y": 5.5, "w": 1.25},
+                { "label": "Space", "matrix": [9, 4], "x": 3.75, "y": 5.5, "w": 6.25},
+                { "label": "Alt", "matrix": [9, 5], "x": 10, "y": 5.5 },
+                { "label": "FN", "matrix": [9, 2], "x": 11.25, "y": 5.5 },
+                { "label": "App", "matrix": [8, 4], "x": 12.5, "y": 5.5 },
+                { "label": "Ctrl", "matrix": [0, 4], "x": 13.75, "y": 5.5 },
+                { "label": "Left", "matrix": [0, 3], "x": 15.25, "y": 5.5 },
+                { "label": "Down", "matrix": [7, 3], "x": 16.25, "y": 5.5 },
+                { "label": "Right", "matrix": [0, 5], "x": 17.25, "y": 5.5 }
+            ]
+        }
+    },
+    "rgb_matrix": {
+        "driver": "AW20216",
+        "animations": {
+            "alphas_mods": true,
+            "gradient_up_down": true,
+            "gradient_left_right": true,
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "rainbow_moving_chevron": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "layout": [ //[row,col]
```

causing closing brace lower down not to recognised
```suggestion
        "layout": [
```